scripts/main.js

```diff
@@ -1,74 +1,90 @@
-            }
-        ],
-    }
+// Check console in Chrome
+console.log("Proces Start");
+
+// // Opdracht 1a: Log het aantal medewerkers van de sales-afdeling in de console, in het volgende format:
+// // "De afdeling Sales heeft [x] medewerkers"
+console.log("De afdeling Sales heeft "+departments.sales.numberOfEmployees+" medewerkers");
+
+// Opdracht 1b: Log de afdeling-beschrijving van de marketingafdeling in de console, in het volgende format:
+// " Marketing is een leuke afdeling om te werken. [beschrijving]."
+console.log("Marketing is een leuke afdeling om te werken. "+departments.marketing.description+".");
+
+// Opdracht 1c: Log het aantal medewerkers van de customer-service-afdeling in de console, in het volgende format:
+// "De afdeling Customer Service heeft [aantal] medewerkers" Let op: er is iets raars aan de hand met deze property!
+console.log("De afdeling Customer Service heeft "+departments["customer-service"].numberOfEmployees+" medewerkers");
+
+// Opdracht 1d: Log de beschrijving van de functie "Verkoopmanager" in de console, in het volgende format:
+// "Sales is een uitdagende afdeling om te werken als Verkoopmanager. [beschrijving functie Verkoopmanager]"
+console.log("Sales is een uitdagende afdeling om te werken als Verkoopmanager. "+departments.sales.jobs[1].description);
+
+// prompt-methode
+// const userInputA = prompt('Hoi! Hoe heet je?');
+// console.log(userInputA);
+
+// Opdracht 2a: Gebruik bovenstaand voorbeeld en pas het zo aan dat de browser jou de volgende vraag stelt:
+// Over welke afdeling wil je meer informatie?
+// Kies uit: [marketing / sales / customer-service].
+// Het antwoord dat jij invoert, log je uiteraard in de console.
+// Tip: geef jouw input-prompt altijd in kleine letters ("marketing" in plaats van "Marketing" of "MARKETING"),
+// dit voorkomt problemen in het script.
+let userInput = prompt('Over welke afdeling wil je meer informatie?\n' +
+    'Kies uit: [marketing / sales / customer-service].');
+console.log(userInput);
+
+// Opdracht 2b:
+// Je koos [ingevoerde]. [beschrijving afdeling]"
+// User filled out info which department
+let chosenInfo;
+switch (userInput) {
+    case "marketing":
+        chosenInfo=departments.marketing.description
+        break;
+    case "sales":
+        chosenInfo=departments.sales.description
+        break;
+    case "customer-service":
+        chosenInfo=departments["customer-service"].description
+        break;
+    case "":
+        userInput="niets";
+        // No break because default had to be handled also (Is this a common or acceptable method?)
```

Comment:
Dit had je nog iets kunnen versimpelen
```suggestion
    case "marketing":
    case "sales":
    case "customer-service":
        chosenInfo = departments[userInput].description
        break;
```